packages/ember/version.d.ts

```diff
@@ -1,2 +0,0 @@
-declare const VERSION: string;
```

Comment:
This type was actually a lie. The generated version module only contains a default export. So this PR is also a bugfix to the types.